build/pkgs/iconv/spkg-check.in

```diff
@@ -1,4 +1,4 @@
-# Only test iconv on Solaris, HP-UX and Cygwin, as those are the only
+# Only test iconv on Solaris, HP-UX, as those are the only
```

Comment:
it's a bit amusing to remove Cygwin but to keep these two (long abandoned) platforms in...

Reply:
as far as I am concerned, we can just as well remove the traces of HP-UX and Solaris here

Reply:
Done in 68c4600 by reducing the whole iconv package to a dummy package.